code/modules/antagonists/dodgeball/dodgeball.dm

```diff
@@ -90,3 +97,11 @@
 		playsound(src, 'sound/items/dodgeball.ogg', 50, 1)
 		visible_message("<span class='danger'>[H] is caught out and explodes!</span>")
 		H.gib()
+
+/obj/item/toy/beach_ball/doomball/afterattack(atom/target, mob/living/carbon/user, proximity)
+	if(!proximity || target == user || !ismob(target) || !iscarbon(user) || user.lying || user.handcuffed) //exploding after touching yourself would be bad
```

Comment:
remove the `|| user.lying || user.handcuffed` part of this check. Players crawling around is nominal, while handcuffed players should already be unable to attack people with objects to start with.